styles.css

```diff
@@ -610,3 +626,486 @@ li {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    color: #172b4d;
+    position: relative;
+    left: 33%;
+    bottom: 10%;
+  }
+
+  .card {
+    width: 380px;
+    height: 498px;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    background: #ebf0ee;
+  }
+
+  .card_inside {
+    width: 380px;
+    height: 238px;
+    align-items: center;
+    position: relative;
+    top: 260px;
+    background: #fff;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    align-content: center;
+  }
+
+  .paragraph_card {
+    width: 337px;
+    height: 57px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 32px;
+    color: #172b4d;
+    padding: 10px;
+    position: relative;
+    margin: 0;
+    padding-top: 45px;
+  }
+
+  .stack {
+    width: 103px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    display: flex;
+    align-items: center;
+    gap: 4px;
+    position: relative;
+    right: 36%;
+    top: 10px;
+    justify-content: space-between;
+  }
+
+  .listw {
+    width: 103px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    justify-content: space-between;
+  }
+
+  .listw li {
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+  }
+
+  #button {
+    position: relative;
+    left: 1%;
+  }
+
+  /* About section */
+
+  .images_about {
+    display: contents;
+  }
+
+  .img_about1 {
+    position: absolute;
+    width: 288.04px;
+    height: 140.24px;
+    bottom: 55%;
+    left: -13%;
+  }
+
+  .img_about2 {
+    position: absolute;
+    width: 269.55px;
+    height: 244.78px;
+    bottom: 60%;
+    right: -13%;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0;
+    gap: 48px;
+    position: absolute;
+    width: 1171px;
+    height: 808px;
+    left: 115px;
+    top: 2300px;
+    background: none;
+  }
+
+  .text_about {
+    padding-left: 34%;
+  }
+
+  .text2_about {
+    width: 920px;
+    height: 96px;
+    font-size: 20px;
+    line-height: 32px;
+    padding-top: 15px;
+    position: relative;
+    left: 30px;
+  }
+
+  #button_resume {
+    position: relative;
+    left: 410px;
+    top: -20px;
+  }
+
+  .skills_container {
+    display: inline-grid;
+    align-items: center;
+    padding: 4px 8px;
+    grid-template-columns: auto auto auto;
+    gap: 24px;
+    width: 1171px;
+    height: 360px;
+    padding-top: 0;
+    margin-top: 0;
+    position: absolute;
+    top: 60%;
+    left: 1.5%;
+  }
+
+  .card_skills {
```

Comment:
Image for Skills and Frameworks is throwing a 404 error in the console of the Mobile and Desktop app. As per the Figma template, your design should look like the below image. Can you please fix this issue? 😃  

Your Design
<img width="375" alt="Screenshot 2023-01-21 at 9 20 17 AM" src="https://user-images.githubusercontent.com/24832846/213844465-22d68c6c-ab51-4c7a-bd46-2349df4a69cd.png">

<img width="1159" alt="Screenshot 2023-01-21 at 9 25 13 AM" src="https://user-images.githubusercontent.com/24832846/213844549-2804f1c2-06ef-426f-a540-acf05ab49f5b.png">


Figma Design
<img width="377" alt="Screenshot 2023-01-21 at 9 20 31 AM" src="https://user-images.githubusercontent.com/24832846/213844476-b01b3643-f2fd-4973-8c57-6d2da316c297.png">

<img width="1208" alt="Screenshot 2023-01-21 at 9 25 24 AM" src="https://user-images.githubusercontent.com/24832846/213844559-989591e0-ebeb-464c-90f3-a6c372ff9be8.png">